src/js/common/reducers/index.js

```diff
@@ -13,5 +15,8 @@ export default combineReducers({
   health: combineReducers({
     rx
   }),
+  disability: combineReducers({
```

Comment:
It seems like this is going to mean our app has all the Redux reducer code for all the other apps in it. That seems bad from a file size perspective. And also possibly cause action name collisions, because we're not trying to make those unique across apps.

Would it be better to have something like a `createCommonReducer` function that took the app specific reducer as a parameter and returned a reducer with the login code added in?